[IMP] doc: add information on the new populate feature. (#50596)
odoo/odoo: doc/reference/cmdline.rst

    a template directory, files are passed through jinja2_ then copied to
    the ``destination`` directory

This will create module *my_module* in directory */addons/*.

.. _reference/cmdline/populate:

Database Population
===================

.. program:: odoo-bin populate

> Odoo CLI supports database population features.  If the feature is implemented on a given model

Reviewer: Double-spacing is illegal.

Author: https://www.theatlantic.com/science/archive/2018/05/two-spaces-after-a-period/559304/

> (see :ref:`Populate API <reference/testing/populate/methods>`), it allows automatic data

Reviewer: Why not just link "implemented on a given model"?

> (See :ref:`Populate API <reference/testing/populate/methods>`), it allows automatic data

Reviewer 2: ```suggestion
(see :ref:`Populate API <reference/testing/populate/methods>`), it allows automatic data
```

> generation of the model's records to easily test your modules in databases containing more records.

Reviewer: ```suggestion
generation of the model's records to test your modules in databases containing non-trivial amounts of records.
```

> generation of the model's records to easily test your modules in heavier or more detailed databases.

Reviewer 2: ```suggestion
generation of the model's records to easily test your modules in heavily loaded databases.
```

Reviewer 3: "heavily loaded"? Or simply "in databases containing more records"

.. code-block:: console

    $ odoo_bin populate

.. option:: --models

    list of models for which the database should be filled

.. option:: --size (small|medium|large)

    population size, the actual records number depends on the model's `_populate_sizes` attribute.
    The generated records content is specified by the :meth:`~odoo.models._populate_factories` method

>     of a given model (cf. the **populate** folder of modules for further details).

Reviewer: ```suggestion
    of a given model (cf. the :file:`populate` folder of modules for further details).
```
